example/ViewPagerAndroidExample.android.js

```diff
@@ -129,25 +142,31 @@ export default class ViewPagerAndroidExample extends React.Component {
     this.setState({page});
   };
 
-  render() {
-    const pages = [];
-    for (let i = 0; i < PAGES; i++) {
-      const pageStyle = {
-        backgroundColor: BGCOLOR[i % BGCOLOR.length],
-        alignItems: 'center',
-        padding: 20,
-      };
-      pages.push(
-        <View key={i} style={pageStyle} collapsable={false}>
-          <Image
-            style={styles.image}
-            source={{uri: IMAGE_URIS[i % BGCOLOR.length]}}
-          />
-          <LikeCount />
-        </View>,
-      );
+  createPage(key) {
+    return {
+            key: key,
```

Comment:
We still need to set up prettier but for now can you fix the indentation manually?